helm/cluster-cloud-director/templates/update-values-hook-job.yaml

```diff
@@ -0,0 +1,153 @@
+# Because RDE ID is not known at the time of cluster creation we use
```

Comment:
@kopiczko We don't need this code after this change. Right?

https://github.com/giantswarm/cluster-apps-operator/blob/master/service/controller/resource/clustersecret/cloud_director.go#L42

Reply:
Yes, the whole `cloud_director.go` should be removed.